src/engine/source/builder/builders/opBuilderHelperActiveResponse.cpp

```diff
@@ -0,0 +1,405 @@
+/* Copyright (C) 2015-2022, Wazuh Inc.
+ * All rights reserved.
+ *
+ */
+
+#include "opBuilderHelperActiveResponse.hpp"
+
+#include <algorithm>
+#include <optional>
+#include <string>
+#include <string_view>
+#include <variant>
+
+#include "syntax.hpp"
+
+#include <baseHelper.hpp>
+#include <utils/socketInterface/unixDatagram.hpp>
+#include <utils/stringUtils.hpp>
+
+using base::utils::socketInterface::SendRetval;
+using base::utils::socketInterface::unixDatagram;
+using helper::base::Parameter;
+
+namespace builder::internals::builders
+{
+
+namespace ar
+{
+
+constexpr const char* AR_JSON_FORMAT {R"({
+                                            "version":0,
+                                            "origin":{
+                                                "name":"",
+                                                "module":""
+                                            },
+                                            "command":"",
+                                            "parameters":{
+                                                "extra_args":[],
+                                                "alert":{}
+                                            }
+                                    })"};
+
+
+// JSON Paths
+constexpr const char* ALERT_PATH {"/parameters/alert"};
+constexpr const char* COMMAND_PATH {"/command"};
+constexpr const char* EXTRA_ARGS_PATH {"/parameters/extra_args"};
+constexpr const char* ORIGIN_MODULE_PATH {"/origin/module"};
+constexpr const char* ORIGIN_NAME_PATH {"/origin/name"};
+constexpr const char* VERSION_PATH {"/version"};
+
+// values
+constexpr const char* MODULE_NAME {"wazuh-engine"};
+constexpr const char* NODE_NAME {"node01"};
+constexpr int VERSION {1};
```

Comment:
Add a TODO, this is a standard protocol, and should change when the API is merged, since it contains the classes to generate request and responses.